src/pixel/sender.js

```diff
@@ -13,25 +16,50 @@ export function PixelSender (liveConnectConfig, onload, presend) {
    * @param {StateWrapper} state
    * @private
    */
-  function _send (state) {
+  function _sendAjax (state) {
+    _sendState(state, 'j', uri => {
```

Comment:
DOn't we need the onload here as well?

Reply:
it was one line below. I moved the bakers' calls into a separate function to make the code a bit more clear. And now the ajax callback calls the "onload" and "callBakers" https://github.com/LiveIntent/live-connect/pull/25/files#diff-51925b852a9fa34aa4d658fa7cd30c4dR22-R23